addons/web_editor/static/src/scss/web_editor.common.scss

```
@@ -735,19 +742,41 @@ blockquote {
             $-btn-primary-border-color: if($-btn-primary-border, o-color($-btn-primary-border), $-btn-primary-color);
             .btn-fill-primary {
                 @include button-variant($-btn-primary-color, $-btn-primary-border-color);
+                @if $-btn-primary-gradient {
+                    background-image: $-btn-primary-gradient;
+                    &:hover {
+                        @extend .btn-custom:hover;
+                    }
```

```suggestion
                    @extend .btn-custom;
```
? :)

Especially since extending compound selector leads to unexpected results in SCSS :grimacing: